### PR TITLE
CLOUDP-84357: Added single namespace support

### PR DESCRIPTION
**Pull request by igor-karpukhin**
### All Submissions:
* Added support for single namespace
* Added catalog build script
* Migrated project from 3-alpha to 3
* Bumped up operator-sdk version to v1.16.0
* Deploy to existing Openshift cluster: CLOUDP-109112

* [X] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?

### CODE REVIEW

File: bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml [hunk 1]
```diff
@@ -64,10 +64,11 @@ metadata:
...
     categories: Database
-    description: The MongoDB Atlas Kubernetes Operator enables easy management of Clusters in MongoDB Atlas
-    operators.operatorframework.io/builder: operator-sdk-v1.4.2
-    operators.operatorframework.io/project_layout: go.kubebuilder.io/v3
-  name: mongodb-atlas-kubernetes.v0.7.0
+    description: The MongoDB Atlas Kubernetes Operator enables easy management of
+      Clusters in MongoDB Atlas
+    operators.operatorframework.io/builder: operator-sdk-v1.16.0
+    operators.operatorframework.io/project_layout: go.kubebuilder.io/v2
+  name: mongodb-atlas-kubernetes.v0.5.0
```
Comment (fabritsius) [inline bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml:71]: This is suspicious, the version shouldn't change ☝️

Comment (igor-karpukhin) [reply bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml]: Fixed. Let's add this folder to `.gitignore` in a separate PR

**Review by fabritsius — APPROVED** [review]
Looks great 👍

File: bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml (hunk 1 above)
Comment (fabritsius) [reply bundle/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml]: Sure

**Pull request merged**